configuration/arm64/initialization.sh

```diff
@@ -11,3 +38,12 @@ sudo apt-get -y install docker-ce docker-ce-cli containerd.io docker-compose-plu
 sudo apt -y install python3-pip
 sudo pip3 install docker-compose
 sudo docker-compose up -d
+
+#create v2xhub user
+cd mysql
+./add_v2xhub_user.bash
+
+sudo apt-get update -y && sudo apt update -y
+
+echo -e "\n open chromium-browser, go to https://127.0.0.1:19760, accept certs, and go to https://127.0.0.1 to login to V2x hub."
```

Comment:
You know what would potentially be a nice to have. Can we launch chromium or the default browser from this script to automatically route to both the certificate page and the login page?

Reply:
It looks like this works `chromium-browser https://www.google.com` for one tab still looking if we can do something for 2 tabs

Reply:
if we run 
chromium-browser "https://127.0.0.1" > /dev/null 2>&1 &
chromium-browser "https://127.0.0.1:19760" > /dev/null 2>&1 &
it opens both tabs, I will update file to have that